### ISSUE TITLE
Add `and et al.` to bibtex export if the number of authors is cut

### ISSUE THREAD

**Issue by golnazads** [adsabs/export_service#134]
```
@ARTICLE{2019ApJ...875L...1E,
   author = {{Event Horizon Telescope Collaboration} and {Akiyama}, K. and 
	{Alberdi}, A. and {Alef}, W. and {Asada}, K. and {Azulay}, R. and 
	{Baczko}, A.-K. and {Ball}, D. and {Balokovi{\'c}}, M. and {Barrett}, J. and et al.},
    title = "{First M87 Event Horizon Telescope Results. I. The Shadow of the Supermassive Black Hole}",
  journal = {\apjl},
archivePrefix = "arXiv",
   eprint = {1906.11238},
 keywords = {accretion, accretion disks, black hole physics, galaxies: active, galaxies: individual: M87, galaxies: jets, gravitation },
     year = 2019,
    month = apr,
   volume = 875,
      eid = {L1},
    pages = {L1},
      doi = {10.3847/2041-8213/ab0ec7},
   adsurl = {https://ui.adsabs.harvard.edu/abs/2019ApJ...875L...1E},
  adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```

**Comment by golnazads** [comment]
also when the number of authors is > 200 cut it to whatever user has specified (num_authors), per Alberto:
aaccomazzi [1:24 PM]
this is the classic code:
```#define BIBTEX_BUFLEN 4096
#define DEF_TRUNC_AUTH 10
    if (extended_bibtex == 0 && authors != NULL && strlen(authors) > BIBTEX_BUFLEN) {
        int nauthors = DEF_TRUNC_AUTH;
...```
So the check is done on the actual length of the author buffer.  If it's greater than a given threshold, we cut the list.  We could do the same in BBB, since what we are trying to avoid is a situation where bibtex barfs due to a buffer overflow.
At an average of 20 characters per author name, we could do the truncation when the author list is above 200 I think

**Comment by golnazads** [comment]
fixed in v2.0.51.
